From 0c682e8793b5478eef42a6a829933142d4ba3721 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 15 Aug 2018 20:14:20 +0800
Subject: [PATCH 081/151] arm64: allwinner: dts: h6: fix the VDD-CPUX voltage
 of ALL-H3-CC H5 ver

The VDD-CPUX voltage of the H5 ver of Libre Computer ALL-H3-CC is not
1.2V but 1.1V.

Fix this in the device tree.

Fixes: af5d05bdc99c("arm64: dts: allwinner: Add dts file for Libre Computer Board ALL-H3-CC H5 ver.")
Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 .../boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts     | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts b/arch/arm64/boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts
index 95e113ce8699..506920a8f38a 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5-libretech-all-h3-cc.dts
@@ -12,3 +12,8 @@
 	model = "Libre Computer Board ALL-H3-CC H5";
 	compatible = "libretech,all-h3-cc-h5", "allwinner,sun50i-h5";
 };
+
+&reg_vdd_cpux {
+	regulator-min-microvolt = <1100000>;
+	regulator-max-microvolt = <1100000>;
+};
-- 
2.18.0

